ingenialink/servo.py

```diff
@@ -284,10 +284,10 @@ def restore_parameters(self, subnode=None):
             # Restore subnode 0
             raise ILError('The current firmware version does not '
                           'have this feature implemented.')
-        elif subnode > 0 and subnode in self.RESTORE_MOCO_ALL_REGISTERS:
+        elif subnode > 0:
             # Restore axis
-            self.write(reg=self.RESTORE_COCO_ALL,
-                       data=self.RESTORE_MOCO_ALL_REGISTERS[subnode],
+            self.write(reg=self.RESTORE_MOCO_ALL_REGISTERS,
+                       data=PASSWORD_RESTORE_ALL,
                        subnode=subnode)
             logger.info(f'Restore subnode {subnode} successfully done.')
         else:
```

Comment:
I would add the subnode value in the error message below.

Reply:
Done!